cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -789,536 +1552,302 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
-														
-		
-													})
+								var custom_field = [
+                                {
+                                    doctype: "Salary Structure Assignment",
+                                    label: rowData.salary_component,
+                                    field_type: "Check"
+                                },
+                                {
+                                    doctype: "Salary Structure Assignment",
+                                    label: rowData.salary_component + " Value",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)